package.json

```diff
@@ -65,10 +68,11 @@
     "postcss": "~8.4.16",
     "postcss-cli": "~10.0.0",
     "prettier": "~2.7.1",
-    "react": "~16.13.1",
+    "react": "~16.14.0",
     "react-bootstrap": "~2.5.0",
```

Comment:
check for all major dependencies like react, redux, bootstrap, etc used here matched with MM v6.0.0, if not then we will need to upgrade/downgrade to match MM